src/opers.h

```diff
@@ -221,17 +222,35 @@ static inline void SET_ENABLED_ATTR(Obj oper, Int x)
 /****************************************************************************
 **
 *F * * * * * * * * * * * * internal flags functions * * * * * * * * * * * * *
+**
+** Attempting change April 2018. Flags will be stored as 
```

Comment:
We could introduce a `struct FlagsHeader` for this, just like we did for other types by now. That's self-documenting and can be used to make some of the accessor functions (well, all but the one for the TRUES_FLAGS) more readable.